File: app/src/main/java/com/simpleshift/scheduler/domain/model/MonthlyStats.kt
package com.simpleshift.scheduler.domain.model

import androidx.compose.runtime.Immutable

@Immutable
data class MonthlyStats(
    val morningCount: Int = 0,
    val afternoonCount: Int = 0,
    val restCount: Int = 0,
    val nightCount: Int = 0,
    val studyCount: Int = 0
)

File: flutter/lib/domain/models/monthly_stats.dart
/// 月度统计 — 对应 Android 版 MonthlyStats.kt
class MonthlyStats {
  final int morningCount;
  final int afternoonCount;
  final int restCount;
  final int nightCount;
  final int studyCount;

  const MonthlyStats({
    this.morningCount = 0,
    this.afternoonCount = 0,
    this.restCount = 0,
    this.nightCount = 0,
    this.studyCount = 0,
  });
}
